src/main/java/org/nuxeo/ecm/csv/CSVImporterDocumentFactory.java

```diff
@@ -36,6 +36,16 @@ public void createDocument(CoreSession session, String parentPath, String name,
 
     public void updateDocument(CoreSession session, DocumentRef docRef, Map<String, Serializable> values);
 
+    /**
+     * @return {@code true} if a document with the specified parentPath and name exists. {@code false} otherwise.
+     * @since 8.1
+     */
+    public boolean exists(CoreSession session, String parentPath, String name);
+
+    /**
+     * @deprecated since 8.1
+     */
+    @Deprecated
```

Comment:
Why deprecating this method and add a new one without `values`?

We may not use `values` in the default implementation but the point here is to allow users creating their own factory to make use of the values, if they want,